### ISSUE TITLE
Fix focus issue with objects text field

### ISSUE THREAD

**Issue by langurmonkey** [langurmonkey/gaiasky#106]
Once the objects text field (in the controls window) captures the keyboard focus never lets it go making it impossible to send keyboard shortcut commands or close the application with escape. This should be fixed asap.
